app/_lib/_db/index.ts

```diff
@@ -83,3 +83,38 @@ export async function storeKarma(
 
   return affectedUsers
 }
+
+interface User {
+  username: string
+  displayName: string
+  realName?: string
+  avatarUrl?: string
+  timezone?: string
+  isBot?: boolean
+  isActive?: boolean
+}
+
+export const getUser = async (provider: string, providerId: string) => {
+  return await prisma.user.findUnique({
+    where: {
+      provider_providerId: {
+        provider: provider,
+        providerId: providerId,
```

Comment:
is providerId the userId?

Reply:
yes it is the userId from the provider in our case slakc ID. But if we use another provider it will be another string. This is like an externalId